.github/workflows/master.yml

```diff
@@ -308,28 +308,67 @@ jobs:
         git config --global user.name "ai2service"
         git config --global push.default simple
 
-    - name: Deploy docs
-      # Only run this on main repo (not forks) for commits and releases but not for
-      # nightly builds.
-      if: github.repository == 'allenai/allennlp' && github.event_name != 'schedule'
+    - name: Set target folders
       run: |
         if [[ $GITHUB_EVENT_NAME == 'release' ]]; then
-            DIRECTORY=${GITHUB_REF#refs/tags/};
+            echo "::set-env name=DOCS_FOLDER::${GITHUB_REF#refs/tags/}"
         else
-            DIRECTORY=master;
+            echo "::set-env name=DOCS_FOLDER::master";
         fi
+        echo "Will deploy docs to $DOCS_FOLDER"
 
+    - name: Stage docs
+      # Only run this on main repo (not forks) for commits and releases but not for
+      # nightly builds.
+      if: github.repository == 'allenai/allennlp' && github.event_name != 'schedule'
+      run: |
         # Checkout allennlp-docs to /allennlp-docs
         git clone git@github.com:allenai/allennlp-docs.git ~/allennlp-docs
         
         # Copy the generated docs to the checked out docs repo
-        rm -rf ~/allennlp-docs/$DIRECTORY/
-        mkdir -p ~/allennlp-docs/$DIRECTORY
-        cp -r site/* ~/allennlp-docs/$DIRECTORY
+        rm -rf ~/allennlp-docs/$DOCS_FOLDER/
+        mkdir -p ~/allennlp-docs/$DOCS_FOLDER
+        cp -r site/* ~/allennlp-docs/$DOCS_FOLDER
 
+    - name: Update shortcuts
+      # Only run this on main repo (not forks) for commits and releases but not for
+      # nightly builds.
+      if: github.repository == 'allenai/allennlp' && github.event_name == 'release'
+      run: |
+        cd ~/allennlp-docs/
+        echo "Updating latest/index.html"
+        mkdir -p latest
+        cat >latest/index.html << EOL
+        <!DOCTYPE html>
+        <html>
+          <head>
+            <meta http-equiv="Refresh" content="0; url=/${DOCS_FOLDER}/" />
+          </head>
+          <body>
+            <p>Please follow <a href="/${DOCS_FOLDER}/">this link</a>.</p>
+          </body>
+        </html>
+        EOL
+        cat latest/index.html
+
+        SUFFIX=$(echo $DOCS_FOLDER | sed -E 's/v[0-9]+\.[0-9]+\.[0-9]+//')
+        if [[ -z $SUFFIX ]] || [[ "$SUFFIX" == *"post"* ]]; then
+            # If there's no suffix to the version or the suffix is a 'post',
```

Comment:
What does it mean for the suffix being a "post"? As in, a post-release? A hot fix?

`"$SUFFIX" == *"post"*`, does that actually work? Won't that just engage the file glob mechanism?

Reply:
I did double-check that it worked, but with the last commit I just pushed I removed this line anyway.

Reply:
Weird bash stuff